### ISSUE TITLE
('NeedsBrowser', 'To access your account, you must sign in on the web. Touch Next to start browser sign-in.')

### ISSUE THREAD

**Issue by chaoscreater** [trustmaster/gkeep2notion#10]
Getting the following error:

```
Loading access token from keyring
Logging into Google Keep
Password:
Authenticating, this may take a while...
Authentication failed
('NeedsBrowser', 'To access your account, you must sign in on the web. Touch Next to start browser sign-in.')
```

I had a working Google Keep export script from another Github project and it's also giving me the same error. Generated new app password code and tried that, same issue.

**Comment by trustmaster** [comment]
Hi @chaoscreater ! Have you tried the methods described in https://github.com/trustmaster/gkeep2notion#preventing-authentication-failed-on-some-systems ?

**Comment by chaoscreater** [comment]
> Hi @chaoscreater ! Have you tried the methods described in [trustmaster/gkeep2notion#preventing-authentication-failed-on-some-systems](https://github.com/trustmaster/gkeep2notion#preventing-authentication-failed-on-some-systems) ?

Yes, tried it already. It's not an app password issue. If it is, then it'll throw a different error.
